### ISSUE TITLE
leaving half-finished posts or list of "active" post ideas?

### ISSUE THREAD

**Issue by nell-byler** [astroDJC/suggestedposts#14]
How long can I leave a half-finished post lying around before it's closed? i.e. if i have an idea for a post and perhaps a list of references for that post but have't gotten around to making said post, is it appropriate to leave it as a .md in the suggested posts folder? It gets hairy when there are many active/half-written posts that are hard to parse through.

Maybe we could have a dedicated wiki page for "active ideas" instead? where we can list a topic and our resources for it, and then anyone can add and edit this list - add a topic, add a resource to another topic etc. Then, when you finish a topic, you can remove it from the list. It also might help the "crowd sourcing" idea because people will know what needs to be covered.

**Comment by bmorris3** [comment]
I think it's OK to start a pull request on an unfinished post as long as you say very clearly that you're still working on it in the pull request comments. Then other people can see your work and suggest changes, without it appearing in the master `merged_posts` dir, where only finished posts should be.

**Comment by nell-byler** [comment]
In the creating the "active topics" page (https://github.com/astroDJC/suggestedposts/wiki/Topics) I stumbled into a content area that I _often_ find myself in when thinking about appropriate material for DJC discussions -- the relevance factor. While there are some topics that can be deemed critical enough to merit a blog post without referencing STEM in particular, this isn't always the case. I just made a blog post on white privilege, which doesn't reference a journal article that specifically references STEM, but I think is an important enough topic it should still be included. BUT when creating the "active topics" section I found myself including things that weren't relevant enough and should just be included as "resources". How do we make that call in the future?

**Comment by bmorris3** [comment]
I think my response is a question: should things that aren't relevant enough be put here at all? I think the purpose of the blog is very specific, and while many related conversations could be had, this might not be the place for them.

**Comment by nell-byler** [comment]
I think that is a good point, and it's a good reminder for us all on ~scope~...but the question remains: how do we determine what is relevant _enough_? If someone submits a blog post that they presented on at DJC at their own institution, but STEM isn't explicitly referenced, who decides if it makes it on the blog?

**Comment by sjschmidt** [comment]
Wait, I'm confused about why STEM needs to be explicitly referenced. Many of the things that we should include are about human interactions or general workplace climate (so on, so forth) and have relevance but not anything direct. 

I do think we should include blog posts (and news articles) in astroDJC, but with the caveat that they should maybe be worthy of an hour of discussion? Maybe default to "blog roundup"?

**Comment by bmorris3** [comment]
I like where this is going. I think the general direction of these comments is that if someone contributes a resource because they think it's important, it's probably important. Should we make a general rule about not rejecting blog posts (unless they violate some common sense rules)?

**Comment by sjschmidt** [comment]
If we're not sure of a post, maybe we should just try to adopt a polite reviewer-type stance "we're not sure what the significance is... can you explain that in your summary"? Otherwise, defaulting to accept seems right...

**Comment by bmorris3** [comment]
:+1: Sounds good to me.
